src/datasource-zabbix/config.controller.js

```diff
@@ -38,6 +32,22 @@ export class ZabbixDSConfigController {
     }
   }
 
+  resetUsername() {
+    this.current.secureJsonFields.username = false;
```

Comment:
I don't think it makes sense to store username in secure json data. Encrypted password is enough.